.travis.yml

```diff
@@ -9,14 +9,23 @@ branches:
   - grails4
   - feature/grails5
   - feature/plugin-upgrade
+  - feature/grails6-upgrade
+
 before_install:
+      - rm -fr $HOME/.gradle/caches
+      - sudo mkdir -p /data/logger-service/ehcache
+      - sudo chmod o+xw /data
+      - sudo chmod o+xw /data/logger-service
+      - sudo chmod o+xw /data/logger-service/ehcache
+
+before_cache:
 - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
 - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
 cache:
   directories:
-  - "$HOME/.m2"
-  - "$HOME/.gradle/caches/"
-  - "$HOME/.gradle/wrapper/"
+  #- "$HOME/.m2"
+  #- "$HOME/.gradle/caches/"
```

Comment:
Rather than making future builds slower, keep these .gradle and .m2 caches. 

To fix the travis timeout add this install block, https://github.com/AtlasOfLivingAustralia/collectory/blob/develop/.travis.yml#L18

Reply:
Fixed